src/objc/smtp/MCOSMTPSession.mm

```diff
@@ -170,6 +170,25 @@ - (MCOSMTPSendOperation *) sendOperationWithData:(NSData *)messageData
     return result;
 }
 
+- (MCOSMTPSendOperation *) sendOperationWithDataFile:(NSString *)messageDataFile
```

Comment:
I'd like to call this method:
- (MCOSMTPSendOperation *) sendOperationWithContentsOfFile:(NSString *)path
  from:(MCOAddress *)from
  recipients:(NSArray *)recipients